docs/Rules/MA0098.md

```diff
@@ -1 +1,135 @@
 # MA0098 - Use indexer instead of LINQ methods
+
+When working with collections that support indexing (such as arrays, `List<T>`, `IList<T>`, or `IReadOnlyList<T>`), using the indexer syntax `[index]` is more efficient than LINQ methods like `ElementAt()`, `First()`, and `Last()`. This rule suggests replacing these LINQ methods with direct indexer access for better performance.
+
+The indexer approach provides direct access to elements without the overhead of LINQ extension methods and enumeration, resulting in faster execution and reduced memory allocations.
+
+## Examples
```

Comment:
This looks very verbose. I think a few examples in the same code block are sufficient. Something like the following:

````c#
var list = new List<int>();

_ = list.ElementAt(5); // non-compliant
_ = list[5];           // compliant

_ = list.First(); // non-compliant
_ = list[0];           // compliant

_ = list.Last(); // non-compliant
_ = list[^1];           // compliant
_ = list[list.Count - 1];  // compliant

_ = list.FirstOrDefault(); // compliant
_ = list.First(x => x > 1); // compliant
````